deploy-os_servers.yml

```diff
@@ -291,6 +291,112 @@
         wait: true
         timeout: "{{ openstack_api_timeout }}"
 ##############################################################################
+# Configure IRODS security group using Openstack API.
+##############################################################################
+    - name: "Create security group for {{ slurm_cluster_name }} IRODs machine."
+      openstack.cloud.security_group:
+        hosts: irods
+        state: present
+        name: "{{ slurm_cluster_name }}_irods"
+        description: |
+                     Security group for security hardened jumphosts bridging the external and internal network.
+                     Allows SSH inbound on both port 22 and 443.
+                     Allows SSH inbound on both port 1247, 1248, 5432 and range: 20000:20199 for IRODs.
```

Comment:
Ports 1248 and 5432 are still in the description, but no longer in the rule list; May get confusing...